cli/src/main.rs

```diff
@@ -62,6 +62,14 @@ fn run() -> Result<(), CliError> {
                 (@arg quiet: -q --quiet "do not display output")
             )
         )
+        (@subcommand health =>
+            (about: "displays information about network health")
+            (@subcommand status =>
+                (about: "displays a node's version, endpoint, node id, and a list \
```

Comment:
Shorten this so it fits in <80 columns. This isn't always possible to achieve w/clap but where we can, we should. Maybe "Display node-level health information".

Reply:
Isn't there a way to provide multi-line documentation without violating clap's line-length restrictions? I am in favor of clear & helpful information over short lines.